ext/intl/breakiterator/rulebasedbreakiterator_methods.cpp

```diff
@@ -159,7 +159,7 @@ U_CFUNC PHP_METHOD(IntlRuleBasedBreakIterator, getRuleStatusVec)
 	ZEND_ASSERT(BREAKITER_ERROR_CODE(bio) == U_BUFFER_OVERFLOW_ERROR);
 	BREAKITER_ERROR_CODE(bio) = U_ZERO_ERROR;
 
-	std::unique_ptr<int32_t[]> rules = std::unique_ptr<int32_t[]>(new int32_t[num_rules]);
+	const std::unique_ptr<int32_t[]> rules = std::unique_ptr<int32_t[]>(new int32_t[num_rules]);
```

Comment:
Use `std::make_unique` while at it

Reply:
![3bkcig](https://github.com/user-attachments/assets/e49a15a5-0822-4c16-8ee8-3ea3e17b8b3f)

Ok then let's leave that